tracer/src/Datadog.Trace/AppSec/AppSecRequestContext.cs

```diff
@@ -47,27 +49,38 @@ internal void CloseWebSpan(TraceTagCollection tags, Span span)
                 }
             }
 
-            if (_raspStackTraces?.Count > 0)
+            if (_wafTimeout > 0)
             {
-                span.SetMetaStruct(StackKey, MetaStructHelper.ObjectToByteArray(_raspStackTraces));
+                span.Tags.SetMetric(Metrics.WafTimeouts, _wafTimeout);
```

Comment:
Is there a reason you're calling `span.Tags` and not using the `tags` object provided? 🤔  How do you know which to use?

Reply:
`Tags` is the `TraceTagCollection` (meta), while the `span.Tags` is the whole `ITags`